src/mmw/js/src/draw/views.js

```diff
@@ -153,16 +155,28 @@ var PlaceMarkerView = Marionette.ItemView.extend({
 
         this.model.disableTools();
         utils.placeMarker(map).then(function(latlng) {
-            App.restApi.getPolygon({
-                shapeType: shapeType,
-                lat: latlng.lat,
-                lng: latlng.lng
-            }).then(function(shape) {
-                addLayer(shape);
-                navigateToAnalyze();
-            }).fail(function() {
-                revertLayer();
-            });
+            // TODO: This is temporary until we have
+            // endpoints that can actually delienate
+            // watersheds.
+            var point = {
+                  "type": "Feature", "properties": {}, "geometry": {
```

Comment:
Ah, yet another reason I dislike the single var statement. It messes up the nesting for objects like this.